src/lib/web-component/log-view/log-view-element.js

```diff
@@ -31,6 +33,23 @@ export class LogView extends HTMLElement {
     for (const row of this.tableBody.rows) {
       row.hidden = !this.isFilterMatched(row._log);
     }
+
+    this.updateRowCountAndStyle();
+  }
+
+  updateRowCountAndStyle() {
```

Comment:
Let's change it to `triggerLogCountChange()`.

I guess that the `AndStyle` suffix was meant to account for future changes to the even-odd appearance.

Given the potential performance impact of the current implementation (see issue 55), I think that it may make sense to remove filtered rows from the document instead.

Reply:
Renamed the method to `triggerLogCountChange`